examples/gameroom/gameroom-app/src/store/api.ts

```diff
@@ -79,7 +79,7 @@ export async function gameroomPropose(
 }
 
 export async function listGamerooms(): Promise<Gameroom[]> {
-  const response = await gameroomAPI.get('/gamerooms');
+  const response = await gameroomAPI.get('/gamerooms?limit=1000');
```

Comment:
Does the server side have a max limit?

Reply:
No, the limit gets passed directly to the database query. I don't see any limit maximum documented from diesel.

Reply:
We should add a maximum on the server-side for this, as well.  Perhaps in a separate commit.